pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1010,6 +1019,44 @@ func getJmsConfig(environment api.EnvironmentType, jms *api.KieAppJmsObject) (*a
 	return jms, nil
 }
 
+func getKafkaConfig(kafka *api.KafkaExtObject) (*api.KafkaExtObject, error) {
+	if kafka == nil || !kafka.ExtEnabled {
+		return nil, nil
+	}
+
+	var override = false
+	defaultKafka := api.KafkaExtObject{}
+	//if something is missing we set mandatory defaults
+	if kafka.MaxBlockMs == 0 {
+		defaultKafka.MaxBlockMs = 2000
+		override = true
+	}
+	if len(kafka.BootstrapServers) == 0 {
+		defaultKafka.BootstrapServers = "localhost:9092"
```

Comment:
I don't think it a good idea to set a default value for the server address, probably better make this field mandatory and fail the kafka setup if it is not set.

Reply:
Is asked here as default value https://issues.redhat.com/browse/RHPAM-3331?focusedCommentId=15539246&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15539246